### ISSUE TITLE
Fix timeout error handling for caps list init

### ISSUE THREAD

**Issue by bit-pirate** [robotics-in-concert/rocon_app_platform#126]
This is a reminder for myself to update the timeout error handling in the caps lists initialisation, once the follow-up changes of https://github.com/osrf/capabilities/pull/39 have been implemented.

**Comment by bit-pirate** [comment]
Currently implemented in 36a65ea9fdac6af5c347945232f6a1ecaf9edcac

Will close this issue once this is merged into hydro-devel.
